src/components/dar_table/DarTable.js

```diff
@@ -121,10 +121,11 @@ export default function DarTable(props) {
   const createElection = async (darId, index) => {
     if (!isNil(darId)) {
       try{
-        const updatedElection = await Election.createDARElection(darId);
-        const votes = await Votes.getDarVotes(darId);
+        const updatedElections = await Election.createDARElection(darId);
+        const electionIds = map(election => election.electionId)(updatedElections);
+        const votes = await Votes.getElectionVotes(darId, electionIds);
```

Comment:
this is why I made the change to the response of createDataRequestElection on the backend 1555 PR
Besides tests, this is the only place the method is called and it is very beneficial to have it return both elections created

Reply:
If it only returned the access election we would have to do another query for the RP election, which we could do if that is preferred to the change in the response but I thought this made more sense

Reply:
API changes like this, by definition, are breaking changes. See https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md#api-changes for more info. New APIs, on the other hand, do not introduce breaking changes and that is the approach we need to take if we want to do something like this.

Reply:
Oh okay, makes sense. So do you recommend I make a new API with this response change and use that one on the UI and file a ticket for the deprecation/deletion of this one?

Reply:
We should discuss implementation in the ticket - PR are an awkward place for this conversation.